system/jlib/jptree.cpp

```diff
@@ -7823,7 +7823,7 @@ static const char * extractOption(const char * option, const char * cur)
 
 static bool ignoreOption(const char * name)
 {
-    return streq(name, "config") || streq(name, "outputconfig");
```

Comment:
This was just a bug preventing outputconfig option from working

Reply:
Why did this prevent the option from working?

Reply:
I don't know.Maybe I needed this change in one WIP version of my PR but I don't think it is needed or correct in its final form. I have put it back the way it was.